### PR TITLE
Remove tags from lyrics

### PR DESCRIPTION
**Pull request by Ritiek**
Prevents tags like `<br>, <p>, </p> ..` etc. from being present in the output lyrics.
(the old code did not seem to work somehow)

### CODE REVIEW

File: instantmusic-0.1/bin/instantmusic [hunk 1]
```diff
@@ -178,13 +178,11 @@ def query_and_download(search, has_prompts=True, is_quiet=False):
             except:
                 print ('Cant get album name')
             try:
+                test1 = ('')
```
Comment (yask123) [inline instantmusic-0.1/bin/instantmusic]: Can we have more verbose variable names ? I know it wasn't much verbose before either, but now is a good opportunity to fix that.

File: instantmusic-0.1/bin/instantmusic [hunk 2]
```diff
@@ -178,13 +178,11 @@ def query_and_download(search, has_prompts=True, is_quiet=False):
...
                 raw_lyrics= (soup.findAll('p', attrs={'class' : 'verse'}))
-                paras=[]
-                test1=unicode.join(u'\n',map(unicode,raw_lyrics))
-
-                test1= (test1.replace('<p class="verse">','\n'))
-                test1= (test1.replace('<br/>',' '))
-                test1 = test1.replace('</p>',' ')
+                for line in raw_lyrics:
+                    test1 = test1 + str(line.get_text()) + '\n'
```
Comment (yask123) [inline instantmusic-0.1/bin/instantmusic]: Same

File: instantmusic-0.1/bin/instantmusic [hunk 3]
```diff
@@ -178,13 +178,11 @@ def query_and_download(search, has_prompts=True, is_quiet=False):
...
+                
                 print (test1)
```
Comment (yask123) [inline instantmusic-0.1/bin/instantmusic]: Can we remove this?

**Comment by Ritiek** [comment]
I've changed the variable names.. Does it look okay now?

**Comment by yask123** [comment]
@Ritiek  Looks much better now. Just one more thing, can you squash all commits into one before I merge this?

**Comment by Ritiek** [comment]
@yask123 I'm not sure but isn't it the author of the repo who chooses this at the time of merge?

**Comment by yask123** [comment]
Yea @Ritiek you are right. Github added this feature lately, I had forgotten about this.

**Pull request merged**